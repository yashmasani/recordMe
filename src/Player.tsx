import React,{FunctionComponent} from 'react';


export const Player:FunctionComponent= () =>{
  
  return(
    <>
      {/** Only shows if there is a file created***/}
      {/** There is a play button, pause button and  **/}

    </>
  )

}

