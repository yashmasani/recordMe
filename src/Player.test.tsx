
import React from 'react';
import '@testing-library/jest-dom';
import {Player} from './Player';
import renderer from 'react-test-renderer'

it('Audio Player renders correctly at start', () => {
  
  const templateString = "";
  const defaultStateRecord = false;

  const tree = renderer
    .create(<Player recordStart={defaultStateRecord} mediaBlobUrl={templateString} />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});

