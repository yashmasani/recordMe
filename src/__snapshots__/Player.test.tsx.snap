// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Audio Player renders correctly at start 1`] = `null`;
